src/btree/bt_handle.c

```diff
@@ -251,12 +245,11 @@ __wt_btree_close(WT_SESSION_IMPL *session)
     F_SET(btree, WT_BTREE_CLOSED);
 
     /*
-     * If closing a tree let sweep drop lookaside entries for it.
+     * Verify the lookaside state.
```

Comment:
A bit more detail about what a valid lookaside state means would be helpful here. The conditional below is fairly complicated.

Reply:
Thanks, I added more detail to this comment.